clients/go/pkg/commands/evaluate_decision.go

```diff
@@ -0,0 +1,119 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+	"github.com/camunda/zeebe/clients/go/v8/internal/utils"
+	"github.com/camunda/zeebe/clients/go/v8/pkg/pb"
+)
+
+type DispatchEvaluateDecisionCommand interface {
+	Send(context.Context) (*pb.EvaluateDecisionResponse, error)
```

Comment:
❓  Why do we need this interface? Can't we just put the send on the step2 interface?

Reply:
I'm not really sure. I used the `create_instance.go` file as an example of what I needed to do. There is a `Dispatch*Command` [there as well](https://github.com/camunda/zeebe/blob/e9b203656e6787d434aff132a8a19b401e581a08/clients/go/pkg/commands/create_instance.go#L26-L32), even though it doesn't seem necessary. It's the same for the [Complete Job command](https://github.com/camunda/zeebe/blob/f831ee5cb54bc14496e19018e4dad83b64d3c9fc/clients/go/pkg/commands/complete_job.go#L24-L26).

Reply:
Okay, let's just stick to what we're doing in other cases then 🤷